layouts/partials/announcement.html

```diff
@@ -24,6 +24,9 @@ <h4>
         </h4>
         {{ end }}
         <p>{{ .message | markdownify }}</p>
+        {{- if eq .dismissable true -}}
+        <button  id="banner-dismiss" class="button">{{ T "banner_acknowledgement" }}</button>
```

Comment:
I'd add this button via JavaScript. A user agent without JavaScript shouldn't see the button.

Reply:
I've set the initial display to none; Only JS users will see the button

Reply:
That sounds like a reasonable compromise.